packages/alfa-dom/test/get-attribute.spec.tsx

```diff
@@ -30,3 +31,44 @@ test("Gets an attribute value when it is defined and lowercase=true", t => {
     "foobar"
   );
 });
+
+test("Gets an attribute with a HTML namespace", t => {
+  const div = <div aria-labelledby="foobar" />;
+
+  t.equal(getAttribute(div, "aria-labelledby", Namespace.HTML), "foobar");
```

Comment:
`getAttribute` should return `null` in the above case; in HTML5, attributes are not assigned no namespaces, not even the HTML namespace.